app/build.gradle

```diff
@@ -40,9 +40,17 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
+    buildFeatures {
+        viewBinding true
```

Comment:
Maybe I missed something, but I didn't see any view binding in the code

Reply:
fixed